server/tests/booking.test.ts

```diff
@@ -0,0 +1,204 @@
+import { format, addDays } from 'date-fns';
+import { configureServer, normalUserEmail, officeQuotas } from './test-utils';
+
+const { app, resetDb } = configureServer('bookings');
+
+beforeAll(resetDb);
+
+describe('Testing DB logic', async () => {
+  test('can create booking and successfully increase booking count', async () => {
+    const office = officeQuotas[0].name;
+    const date = format(addDays(new Date(), 1), 'yyyy-MM-dd');
+    const createBookingBody = {
+      user: normalUserEmail,
+      office,
+      date,
+      parking: false,
+    };
+    const createResponse = await app
+      .post('/api/bookings')
+      .send(createBookingBody)
+      .set('bearer', normalUserEmail);
+
+    expect(createResponse.ok).toBe(true);
+    expect(Object.keys(createResponse.body)).toEqual([
+      'id',
+      'created',
+      'user',
+      'date',
+      'office',
+      'lastCancellation',
+      'parking',
+    ]);
+    expect(typeof createResponse.body?.id).toBe('string');
+    expect(createResponse.body).toMatchObject(createBookingBody);
+
+    const getCreatedBookingResponse = await app
+      .get(`/api/bookings?user=${normalUserEmail}`)
+      .set('bearer', normalUserEmail);
+    expect(getCreatedBookingResponse.body).toContainEqual(createResponse.body);
+
+    const getOfficeBookingsResponse = await app.get(`/api/offices`).set('bearer', normalUserEmail);
+    const officeData = getOfficeBookingsResponse.body.find((item: any) => item.name === office);
+    const slot = officeData.slots.find((item: any) => item.date === date);
+    expect(slot.booked).toEqual(1);
```

Comment:
If we're creating without parking, we should assert that the parking count has not increased.

Reply:
We're now using isolated users for each test to avoid conflicts instead